test/tests.js

```diff
@@ -999,4 +999,33 @@ describe('pasync', function() {
 			done();
 		}).catch(done);
 	});
+
+	it('times', function(done) {
+		var timesCount = 0;
+		var timesTest = 2;
+		var times = pasync.times(8, function(n, next) {
```

Comment:
Why are you assigning this to a variable?  (ie, the variable called 'times')  It's not being used.